modules/_labs/menu/react/stories/stories_visualTesting.tsx

```diff
@@ -33,11 +33,20 @@ const createMenuItems = (hasIcons?: boolean, isFocused?: boolean): StoryMenuItem
       isFocused: isFocused,
     },
     {
-      text: `Second Item with really really really long label`,
+      text: `Second Item (with a really really really long label)`,
       icon: undefined,
     },
     {
-      text: `Third Item (disabled)`,
+      text: `Third Item (with a really really really long label)`,
+      icon: hasIcons ? uploadCloudIcon : undefined,
+    },
+    {
+      text: `Fourth Item (with a really really really long label)`,
+      icon: hasIcons ? uploadCloudIcon : undefined,
+      secondaryIcon: hasIcons ? extLinkIcon : undefined,
+    },
```

Comment:
Part of the reason we missed this (I think) is because we didn't have a visual test for it. Though we have a story that shows the issue, they aren't inspected as closely as the visual regression tests.